enterprise/frontend/src/metabase-enterprise/whitelabel/components/HelpLinkSettings/HelpLinkRadio.tsx

```diff
@@ -0,0 +1,62 @@
+import { t } from "ttag";
+import { useCallback } from "react";
+import { Radio, Stack } from "metabase/ui";
+
+interface Props {
+  setting: { value: string };
```

Comment:
what do you think about typing this as an enum? Since the there are only 3 options here.

https://github.com/metabase/metabase/blob/08480aac9416ea03ba5ae525f67b27da2ea54112/frontend/src/metabase-types/api/settings.ts#L190